Client/lib/gMap.js

```diff
@@ -0,0 +1,231 @@
+
+var gMap = function(){
+
+  getGeolocation(startGMap);
```

Comment:
I like the functions at the bottom of the file and with that style I define my function as function name which get hoisted in their completely to the top rather then just the variable name.